### PR TITLE
[ADD] book resources for the whole day

### PR DESCRIPTION
**Pull request by robinkeunen**
- add resource_allocation_start, resource_allocation_end
  and set_allocation_span fields
- compute default values on these fields based on
  'date_start', 'date_end', 'need_delivery', 'delivery_time',
  'pickup_time', 'set_allocation_span'

### CODE REVIEW

File: resource_activity/models/resource_activity.py [hunk 1]
```diff
@@ -129,11 +141,50 @@ def onchange_location_id(self):
...
+            if self.need_delivery:
+                if self.set_allocation_span:
+                    end = _pd(self.date_end) + timedelta(minutes=90)
+                else:
+                    end = self.pickup_time if self.pickup_time else self.date_end
+                    end = self._localize(_pd(end))
+                    end = trunc_day(end) + timedelta(days=1)
+                    end = end.astimezone(pytz.utc)
+            else:
+                end = _pd(self.date_end)
```
Comment (houssine78) [inline resource_activity/models/resource_activity.py]: two nearly same code blocs could go in a generic function?

File: resource_activity/models/resource_activity.py [hunk 2]
```diff
@@ -356,7 +407,7 @@ def write(self,vals):
     @api.multi
     @api.depends('partner_id', 'registrations_max','registrations_expected')
     def _propagate_activity_fields_update(self):
-        for activity in self:
+        for activity in self:  # fixme
             vals['partner_id'] = activit.partner_id
```
Comment (houssine78) [inline resource_activity/models/resource_activity.py:407]: what needs to be fixed?

**Review by houssine78 — APPROVED** [review]
small remarks but approved

File: resource_activity/models/resource_activity.py (hunk 2 above)
Comment (robinkeunen) [reply resource_activity/models/resource_activity.py:407]: cf https://github.com/houssine78/addons/pull/15

Comment (houssine78) [reply resource_activity/models/resource_activity.py:407]: #15 doesn't say what need to be fixed

Comment (houssine78) [reply resource_activity/models/resource_activity.py:407]: i've closed it by the way.

**Pull request merged**